src/lib/webext.ts

```diff
@@ -65,6 +65,14 @@ export async function activeTabContainerId() {
     return (await activeTab()).cookieStoreId
 }
 
+export async function getTabsByUrl(url: string) {
+    return await browserBg.tabs.query({url});
+}
+
+export async function getTabByTabId(tabId: number) {
+    return await browserBg.tabs.get(tabId);
```

Comment:
These functions are so simple that it's probably not worth having them - I'd just use browserBg directly.

Reply:
Funcions removed